lib/ansible/modules/crypto/openssl_certificate.py

```diff
@@ -0,0 +1,760 @@
+#!/usr/bin/python
```

Comment:
This seems to be the convention in the Ansible code base [1][2][3]

[1] http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_general.html
[2] https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/files/archive.py
[3] https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/files/assemble.py